src/httpserver.cpp

```diff
@@ -449,6 +420,7 @@ bool UpdateHTTPServerLogging(bool enable) {
 
 std::thread threadHTTP;
 std::future<bool> threadResult;
```

Comment:
Nit, this and the above could be static too.

Reply:
Yes, but out of scope of this PR to change that.